R/inferCNV.R

```diff
@@ -24,8 +24,12 @@
 #' @slot reference_grouped_cell_indices <list>  mapping [['group_name']] to c(cell column indices) for reference (normal) cells
 #'
 #' @slot observation_grouped_cell_indices <list> mapping [['group_name']] to c(cell column indices) for observation (tumor) cells
+#' 
+#' @slot tumor_subclusters <list> stores subclustering of tumors if requested
 #'
-#' @export
+#' @slot hspike
+#' 
+#' @export .hspike a hidden infercnv object populated with simulated spiked-in data
```

Comment:
The hspike slot description is on the wrong tag

Reply:
I'll blame my old keyboard.   New keyboard works awesome. :-)